packages/ui5-info/src/ui5-theme-info.ts

```diff
@@ -85,8 +86,13 @@ function isSupported(theme: UI5Theme, cleanSemVer: SemVer): boolean {
  * @returns UI5 themes array
  */
 export function getUi5Themes(ui5Version: string = defaultVersion): UI5Theme[] {
-    const ui5VersionSince = ui5Version.replace('snapshot-', '');
-    const cleanSemVer = coerce(ui5VersionSince);
+    // Handle 'Latest' versions by using the latest supported UI5 version
+    const isUsingDefaultVersion = ui5Version === defaultVersion;
+    const resolvedUi5Version = isUsingDefaultVersion
+        ? supportedUi5VersionFallbacks[0].version
```

Comment:
Do we now use the fallback also for cases where we would have access to the real data at https://ui5.sap.com/versionoverview.json? My understanding was, to use fallbacks in cases we are not able to read the live data, only.